### ISSUE TITLE
Code Review Summary 2 by Minghe Ren

### ISSUE THREAD

**Issue by rmhsawyer** [Arshitha/Entertainment-Tracking-system#19]
I am glad to see that you have resolved most of the issues that I had bought up previously. There are still a few issues that I found necessary to be rectified.

1. when I type starwar in your search engine, it shows that, here I quota, "Internal Server Error". When I type star war correctly, it gives me the right result.
 I think you can fix this misunderstanding guide page easily by giving the result of "star war" or "invalid input" when I type "starwar".

2. In your front-end and back-end folder, it's still difficult to understand what's in the file because there is no file or code description. And there are some codes contains no comments to tell user what's these files for. To be honest, I am no very familiar with web design, so I suggest you could make more description in these folders to explain your different modules and your hierarchy.

3. The basic functions on your website have been realized successfully, but the core idea of your project, keeping track is still undone.

I hope to see the core idea of your project been realized on your website in next sprint or post presentation, which is very attractive to users.

4. In my playlist, can't enter the cast or review information. There is no response.

5. I notice that there is a same comments for different functions you defined in your Entertainment-Tracking-system/BackEnd/app.js line 55 and 60.

6. I can add a new movie in my playlist, but how can I delete it when I finished it?

7. Why there are two version of your frontend website? You should explain that.

In a nutshell, I think your project is very interesting and you have designed a beautiful webpage. I am looking forward to see your finished project during the poster presentation. Good luck with you!

**Comment by Arshitha** [comment]
Hi, 

1. (reply to 1) We have made that fix but we are yet to integrate it into our main website. We'll do it by sprint 4 and deploy it. 

3. (reply to 3) Working on it. 

4. (reply to 4)This is done. Just needs to be deployed.

6. (reply to 6) Thank you for this suggestion, we hadn't thought about this user story. We will implement asap.

 7. (reply to 7) Initially, one team member built a front-end and later another team member re did it because of some miscommunication among ourselves. We have two front-ends so that during evaluation both members contributions are recognized.
